src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -556,11 +556,15 @@ CreateDistributedTable(Oid relationId, Var *distributionColumn, char distributio
 	{
 		List *partitionList = PartitionList(relationId);
 		Oid partitionRelationId = InvalidOid;
+		Oid namespaceId = get_rel_namespace(relationId);
+		char *schemaName = get_namespace_name(namespaceId);
+		char *relationName = get_rel_name(relationId);
+		char *parentRelationName = quote_qualified_identifier(schemaName, relationName);
 		foreach_oid(partitionRelationId, partitionList)
```

Comment:
wouldn't mind an empty line here

Reply:
I felt so, but I wasn't sure about it and kept the way it is. Will add.